content/en/actions/datastores/trigger.md

```diff
@@ -0,0 +1,52 @@
+---
+title: Use Automation Rules 
+description: Reduce manual effort by triggering workflows automatically when datastore records are created, updated, or change status. 
+disable_toc: false
+aliases:
+- actions/datastore/trigger
+further_reading:
+- link: "https://www.datadoghq.com/blog/datadog-datastore/"
+  tag: "Blog"
+  text: "Enhance your automated workflows and apps with Datastore"
+---
+
+You can add workflows to your datastores that automatically trigger when records are created, updated, or change status.
```

Comment:
I'm not sure what "change status" means, but I think it might be more accurate to say:

```suggestion
You can add workflows to your datastores that automatically trigger when records are created, updated, deleted
```